libs/shared/data-context/src/lib/shared-data-context.tsx

```diff
@@ -1,18 +1,18 @@
-import { createContext, useState, useCallback } from 'react';
+import { createContext, ReactNode, useState } from 'react';
 
-type CounterContextType = {
+export type CounterContextType = {
   counter: number;
   increment: () => void;
   decrement: () => void;
 };
 
 export const DataContext = createContext<CounterContextType>({
   counter: 1,
-  increment: () => {},
-  decrement: () => {},
+  increment: () => null,
+  decrement: () => null,
 });
 
-export const DataProvider = ({ children }: { children: React.ReactNode }) => {
+export const DataProvider = ({ children }: { children: ReactNode }) => {
```

Comment:
This is just a very minor thing for readability. Don't worry about it too much :)